fastestimator/dataset/data/pascal_voc.py

```
@@ -0,0 +1,294 @@
+        """Add Bounding boxes data to a data dictionary.
+
+        Args:
+            filename: The filename of the image for which to find data.
+        """
+        if self.box_images.get(filename) is None:
+            return None, [], False
+
+        image = cv2.imread(self.box_images[filename])
+        image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+
+        target = self.parse_voc_xml(ET_parse(self.box_targets[filename]).getroot())
+
+        # TO-DO
+        # The following function can also be used to output pose for each bbox
+        bbox = self.get_objects(target)
+
+        return image, bbox, True
+
+    @staticmethod
+    def parse_voc_xml(node) -> Dict[str, Any]:
+        '''
+        Parse xml file to read Bounding Box annotations of individual image files
+        '''
+        voc_dict = {}
+        children = list(node)
+        if children:
+            def_dic = collections.defaultdict(list)
+            for dc in map(PascalVoc.parse_voc_xml, children):
+                for ind, v in dc.items():
+                    def_dic[ind].append(v)
+            if node.tag == "annotation":
+                def_dic["object"] = [def_dic["object"]]
+            voc_dict = {node.tag: {ind: v[0] if len(v) == 1 else v for ind, v in def_dic.items()}}
+        if node.text:
+            text = node.text.strip()
+            if not children:
+                voc_dict[node.tag] = text
+        return voc_dict
+
+    def get_objects(self, ann_dict):
+        '''
+        Fetch and rearange Bounding box coordinates
+        '''
+
+        objects = ann_dict['annotation']['object']
+        bbox = []
+        for obj in objects:
+            xmin, ymin, xmax, ymax = obj['bndbox']['xmin'], obj['bndbox']['ymin'], obj['bndbox']['xmax'], obj['bndbox']['ymax']
+            cid = self.cls2ind[obj['name']]
+            bbox.append([xmin, ymin, xmax, ymax, cid])
+
+        return bbox
+
+    def _populate_mask_data(self, filename: str) -> None:
```

default mask should be None